bigml/ensemble.php

```diff
@@ -356,7 +357,7 @@ function predict_probability($input_data, $by_name=true, $method=MultiVote::PROB
          //                 respectively.  If True, returns a list of probabilities
          //                 ordered by the sorted order of the class names.
       if ($this->boosting) {
-          throw new Exception("This function is not implemented for boosting yet.");
+          return $this->boostedensemble->predict($input_data, $by_name, $missing_strategy, $compact);
       }
```

Comment:
This is the place where you can see the other half of the mismatch I mentioned above: this method passes $compact, which is an input par, but it only supports it having either true or false value and false is the default. In other words, I do not see how I could get a prediction without probability (which would correspond to passing null for $compact).